add OF macro to work with recent zlib, The OF macro appears to have changed to _Z_OF in recent zlib.


--- src/dynlibs-orig.h	2021-02-22 16:10:59.953402591 +0000
+++ src/dynlibs.h	2021-02-22 16:11:58.780680049 +0000
@@ -64,6 +64,7 @@
 #else
 extern void * gz_lib;
 #endif
+#define OF(x) x
 extern gzFile (*gzdopen_p)(int, const char *);
 extern int (*gzclose_p)(gzFile);
 extern int (*gzread_p)(gzFile, void*, unsigned);
